### ISSUE TITLE
output plugin base `workers` docs are confusing

### ISSUE THREAD

**Issue by jordansissel** [elastic/logstash#7512]
![image](https://user-images.githubusercontent.com/131818/27406686-835b4dd2-568a-11e7-8de8-1f00493fb3db.png)

The text in this is incomplete.

Recommendations

* For 6.0, Remove the `workers` setting (something we intended to do anyways, and now may be a good time).
* For 5.x, fix the `workers` docs entries.

**Comment by dedemorton** [comment]
@jordansissel This will be fixed with the new plugin doc gen process, no? I've already fixed this multiple times, but my fixes keep getting overwritten when we regenerate the plugin docs.

**Comment by PhaedrusTheGreek** [comment]
When starting LS 5.2.1 with an elasticsearch output `workers` setting, LS outputs this:

```
[2017-10-10T16:58:51,699][ERROR][logstash.agent           ] Cannot create pipeline {:reason=>"You are using a plugin that doesn't support workers but have set the workers value explicitly! This plugin uses the shared and doesn't need this option"}
```

So it seems that LS 5.x doesn't support the workers setting at all - is there a reason why it should remain in the docs?

Also [the current performance troubleshooting guide](https://www.elastic.co/guide/en/logstash/current/performance-troubleshooting.html) explicitly references the `workers` output setting

**Comment by dedemorton** [comment]
Also see this related PR: https://github.com/elastic/logstash/issues/7884

My understanding here is that the setting pipeline.output.workers needs to be removed, but pipeline.workers is still valid. @jordansissel Is that correct?

**Comment by jordansissel** [comment]
Correct.

**Comment by dedemorton** [comment]
Closed by #8562
